frontend/src/components/resourceMap/sources/definitions/NetworkSource.tsx

```diff
@@ -0,0 +1,167 @@
+import { Icon } from '@iconify/react';
```

Comment:
It would be good to have what these sources are documented somewhere.

Reply:
I'd like to document things like this in our docs as well, when we get plugin api for this ready

Reply:
I'd like things documented so I can review more easily.

Reply:
I understand, I was talking about additional documentation on the website for plugin authors

Reply:
added better docs to the GraphSource interface